.github/workflows/cli-release.yml

```
@@ -0,0 +1,35 @@
+name: CLI Release
+
+run-name: CLI Release from ${{ github.ref_name }}${{ github.event.inputs.dry_run == 'true' && '(dry-run)' || '' }}
```

actionlint (1.7.12)

[error] property "dry_run" is not defined in object type {}

(expression)